Backend/src/main/java/se/backend/controller/internal/DogsController.java

```diff
@@ -76,7 +76,7 @@ public DogsController(LostDogService lostDogService, LoginService loginService)
                                                         @RequestPart("dog") LostDogWithBehaviors newDog,
                                                         @RequestPart("picture") MultipartFile picture) {
         logHeaders(headers);
-        if(!loginService.IsAuthorized(headers, List.of(UserType.Regular))) {
+        if(!loginService.IsAuthorized(headers, List.of(UserType.Regular,))) {
```

Comment:
Does it compile? Tests are not passed and the error is that something in this file cannot be compiled

Reply:
Yes this error is fixed. I guess I didn't push it. But I have a different issue with the test in the dog controller that tests the updateDog function. I am still debugging it.